vagrant/values.yaml

```diff
@@ -29,3 +29,31 @@ sumologic:
 telegraf-operator:
   enabled: true
   replicaCount: 1
+
+falco:
+  enabled: true
+  customRules:
+    # Mark the following as known k8s api callers:
+    # * fluentd and its plugins
+    # * grafana sidecar
+    # * terraform provider started in setup job
+    # * telegraf operator
+    # * kube state metrics server
+    # * prometheus
+    # * prometheus operator
+    rules_user_known_k8s_api_callers.yaml: |-
+      - macro: user_known_contact_k8s_api_server_activities
+        condition: >
+          (proc.pcmdline = "fluentd /usr/local/bundle/bin/fluentd -c /fluentd/etc/fluent.conf -p /fluentd/plugins") or
```

Comment:
Can we use both proc names and the namespace?

Reply:
We could but we'd need to template it because user can configure the destination template for our collection?

Reply:
For vagrant env for now hardcoding **should** be fine though